readme.md

```diff
@@ -0,0 +1,77 @@
+### Important: This repository is in an early development phase and not suitable for practical workloads. It does not compare with `docker build` features yet.
+
+
+## Buildkit
+
+Buildkit is a toolkit for converting source code to build artifacts in an efficient, expressive and repeatable manner.
+
+Key features:
+- Automatic garbage collection
+- Extendable frontend formats
+- Concurrent dependency resolution
+- Efficient instruction caching
+- Build cache import/export
+- Nested build job invocations
+- Distributable workers
+- Multiple output formats
+- Pluggable architecture
+
+
+Read the proposal from https://github.com/moby/moby/issues/32925
+
+#### Quick start
+
+[![asciicinema example](https://asciinema.org/a/gPEIEo1NzmDTUu2bEPsUboqmU.png)](https://asciinema.org/a/gPEIEo1NzmDTUu2bEPsUboqmU)
+
+Buildkit daemon can be built in two different versions: one that uses [containerd](https://github.com/containerd/containerd) for execution and distribution and a standalone version that doesn't have other dependencies apart from [runc](https://github.com/opencontainers/runc). We are open for adding more backends. `buildd` is a CLI utility for running the gRPC API. 
```

Comment:
"for execution and distribution and a standalone" -> "for execution and distribution, and a standalone" (add comma before the last "and")